crates/cxx-qt-build/src/lib.rs

```diff
@@ -806,21 +816,39 @@ impl CxxQtBuilder {
         }
     }
 
+    fn clean_directory(path: impl AsRef<Path>) {
+        let result = std::fs::remove_dir_all(&path);
+        if let Err(err) = result {
+            // If the path doesn't exist that's fine, otherwise we want to panic
+            if err.kind() != ErrorKind::NotFound {
+                panic!("Could not delete export directory: {}", err);
```

Comment:
export dir? isn't this a generic path, and same on the expect below

Reply:
As in would it not be better to just say `Could not delete {path}: {err}`

Reply:
Ah, right, maybe I should just bubble up the error.

Reply:
Done, and also moved into the `dir` module.